libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/package.scala

```diff
@@ -5,4 +5,8 @@ package com.daml
 
 package object logging {
   type LoggingContextOf[+P] = LoggingContextOf.Module.Instance.T[P]
+
+  type LoggingKey = String
+  type LoggingValue = String
+  type LoggingEntry = (LoggingKey, LoggingValue)
```

Comment:
Type aliases are types lying about being type safe.

Reply:
The next PR replaces `LoggingValue` and `LoggingEntry` with real classes. :slightly_smiling_face:

Reply:
fine for me then

Reply:
> Type aliases are types lying about being type safe.

just be careful